src/box/lua/config/utils/log.lua

```diff
@@ -0,0 +1,135 @@
+-- Logger wrapper with a few enhancements.
+--
+-- 1. Encode tables into JSON.
+-- 2. Enable all the messages on TT_CONFIG_DEBUG=1.
+--
+-- Hopefully, the wrapper can be dropped when the following logger
+-- problems will be solved.
+--
+-- * JSONify tables at logging (gh-8611)
+-- * Follow TT_LOG_LEVEL before log.cfg()/box.cfg() (gh-8092)
+--
+-- A solution of the following problems may simplify the code of
+-- this wrapper.
+--
+-- * mylog.cfg() configures the main logger (gh-8610)
+-- * No easy way to determine if a message should go to the log
+--   (gh-8730)
```

Comment:
Just to don't try to make the hack look like a general solution (it isn't) and save some time. I would need to solve the following things:

* Accept the logger name. It means that I can't just use it as a module, but should implement .new() function that creates an object with the logger name inside.
* Possibly handle table with `__tostring` and/or `__serialize` somehow specifically. I didn't think in this direction.

And I would prefer to solve the problems right in the logger in the scope of the referred issues instead of paying time to a temporary, but general solution.

Reply:
ok, thanks for explanation